src/scss/pages/_profile.scss

```
@@ -0,0 +1,201 @@
+@use "../helpers/variables" as v;
+
+.profile-container {
+  display: flex;
+  height: 100vh;
```

Jest z tym pewien problem, bo Header zajmuje pewną część vh (viewport height) i przez to pokazuje się scroll pionowy np. u mnie na komputerze na Chromie. bo 100vh + wysokość Headera wywołuje overflow-y. 

Nie wiem jakie byłoby najlepsze rozwiązanie, ale tak tymczasowo po prostu odjąłbym wysokość Headera, czyli:
```suggestion
  height: calc(100vh - 64px);
```
Wtedy powinno być spoko na każdym urządzeniu, bo wysokość Headera jest stała.